module-4/backend/week_one.md

```diff
@@ -5,16 +5,38 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
++ Getting a good understanding of JS syntax. Probably could have frontloaded (optionally?) some of the work from this week into the intermission
 2. What are some tools to help debug JavaScript code?
++ Debugger for client side
++ Pry for node/server side
++ Head scratchingly bad error messages
 3. What are some tools you need in order to unit test your JavaScript?
++ Node Mocha Chai. 
++ A far better understanding of asynchronous JS. 
 4. What is the syntax for invoking a function?
++ define with `function funcName(args){ do the things when called;}; funcName`
++ unnamed define and call with `function(args){ the things to do right now;}`
 5. What's the difference between `==` and `===` in JavaScript?
++ == will do type conversions automatically so ` 4 == '4' == true ` => true, === is strict with types
 6. What's the difference between asynchronous and synchronous JavaScript? 
++ Synchronous will act like ruby and execute in the order it is called. Async uses black magic to do things at the same time, and ruin the intention of your code
```

Comment:
It's *kind of* like "the same time", but remember that it's not running in parallel. It's more like saying "Hey, let me come back to this once it's finished"